stdpopsim/catalog/StrAga/genome_data.py

```diff
@@ -1,6 +1,6 @@
 # File autogenerated from Ensembl REST API. Do not edit.
 data = {
-    "assembly_accession": "GCA_000689235.1",
-    "assembly_name": "GBCO_p1",
-    "chromosomes": {"1": {"length": 2065074, "synonyms": ["I"]}},
+    "assembly_accession": "GCA_001017915.1",
+    "assembly_name": "ASM101791v1",
+    "chromosomes": {},
```

Comment:
another empty one?

Reply:
yep - the current [build for this](http://rest.ensembl.org/info/assembly/streptococcus_agalactiae_GCA_001017915?synonyms=1) has no chromosome-level assembly